lib/src/ui/movie_list.dart

```diff
@@ -1,4 +1,7 @@
 import 'package:flutter/material.dart';
+import 'movie_image.dart';
+import 'movie_detail.dart';
+import '../utils/styles.dart';
 import '../utils/numeric_constants.dart';
```

Comment:
MovieListState should be private to this file

Reply:
Fixed